bilby/gw/likelihood.py

```diff
@@ -1493,15 +1493,16 @@ def setup_bins(self):
             d_phi = np.sum(np.array([np.sign(gamma[i]) * d_alpha[i] * (
                 frequency_array_useful ** gamma[i]) for i in range(len(gamma))]), axis=0)
             d_phi_from_start = d_phi - d_phi[0]
-            num_bins = int(d_phi_from_start[-1] // self.epsilon)
+            self.number_of_bins = int(d_phi_from_start[-1] // self.epsilon)
             self.bin_freqs[interferometer.name] = np.array([frequency_array_useful[np.where(d_phi_from_start >= (
-                (i / num_bins) * d_phi_from_start[-1]))[0][0]] for i in range(
-                    num_bins + 1)])
+                (i / self.number_of_bins) * d_phi_from_start[-1]))[0][0]] for i in range(
```

Comment:
This appears to be different from what the IAS code does. I'm guessing that's why we are required to multiply by a factor while computing the summary data.

Reply:
I did change the `setup_bins` to look exactly like the one in IAS code. The factor of #bins is still creeping in.

Reply:
No I don't think the problem lies here. This is only calculating the bins and setting the bin frequencies.